### PR TITLE
Fix SortMeRNA for Blast output

### PR DESCRIPTION
**Pull request by bebatut**
I got error when running SortMeRNA with Blast output, due to the new SortMeRNA version (changes in options). So I tried to fix that

I also add missing inputs in current tests

### CODE REVIEW

File: tools/rna_tools/sortmerna/sortmerna.xml [hunk 1]
```diff
@@ -75,7 +75,9 @@ sortmerna --version 2>&1|grep 'SortMeRNA version'
     	#if $aligned_sam.aligned_sam_selector == '--sam'
         	$aligned_sam.sq
     	#end if
-    	$aligned_blast
+        #if $aligned_blast.aligned_blast_selector == '--blast'
```
Comment (nsoranzo) [inline tools/rna_tools/sortmerna/sortmerna.xml]: The indentation of this line is a bit off.

File: tools/rna_tools/sortmerna/sortmerna.xml [hunk 2]
```diff
@@ -247,10 +258,7 @@ sortmerna --version 2>&1|grep 'SortMeRNA version'
         </data>
         <data format="tabular" name="output_blast" from_work_dir="aligned.blast"
             label="Alignments on ${on_string} (BLAST)">
-            <filter>aligned_blast</filter>
-            <change_format>
-                <when input="aligned_blast" value="--blast 0" format="txt" />
-            </change_format>
```
Comment (nsoranzo) [inline tools/rna_tools/sortmerna/sortmerna.xml]: Is the `<change_format>` not necessary any more?

Comment (bebatut) [inline tools/rna_tools/sortmerna/sortmerna.xml]: My bad. I remove it too quickly.
I fix it.
Thanks :)

**Comment by nsoranzo** [comment]
Can you increase the tool version number to 2.1b.1 ?

**Comment by bebatut** [comment]
@nsoranzo I did it

**Comment by nsoranzo** [comment]
Thanks @bebatut!

**Comment by bebatut** [comment]
Can I push it on TTS and MTS?

**Comment by bgruening** [comment]
Yes please :)

**Comment by bebatut** [comment]
Done !

**Pull request merged**